pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -57,6 +58,11 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	fmt.Printf("Latest Kubernetes version: %s\n", latestVersion)
 	fmt.Printf("Current Node version: %s\n", nodeVersionInfoUpdate.Current.KubeletVersion.String())
 	fmt.Println()
+	target.IsSUSEOS()
```

Comment:
I would add a comment above L61 to say:
```
// Populate os release version in target Cache to execute OS checks
```

It's not 100% clear the link between 61 and 62.

Reply:
That's a good idea. Thanks.